specs/deployment-repo-update.md

```diff
@@ -0,0 +1,31 @@
+# Deployment Repo Update Automation
+
+## Summary
+
+The GitOps driven continuous deployment cycle starts with a change in the Git repository that contains resource manifests. The [Flux] provides the
+[Automated Image Update](https://docs.fluxcd.io/en/latest/references/automated-image-update.html) feature that continuously monitors the docker registry and automatically
+updates deployment repo when a new image is released. Obviously that functionality is not available for  Argo CD users.
+Also cannot or don't want to use docker-registry monitoring and only need functionality related to the Git repository update.
```

Comment:
I don't really understand this phrase (sorry). Would you mind clarifying it?

Reply:
Now I don't understand it either :) rewritten.